### PR TITLE
Vitty Extension 2.0

### PR DESCRIPTION
**Pull request by KrishChatterjie**
- Vitty Extension V2
- React extension
- No uploading
- Same functionality as the apps
- Clean code :eyes:

### CODE REVIEW

File: src/components/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          <h3>Profile</h3>
+          <FaTimes onClick={onClose} />
+        </div>
+        <div className='modal-body'>
+          <div className='modal-message'><span>Name:</span> {name}</div>
+          <div className='modal-message'><span>Email:</span> {email}</div>
+          <div className='modal-buttons'>
+            <button className='modal-yes' onClick={onLogOut}>Log Out</button>
+          </div>
+          <div className='modal-tip'>Tip: To edit timetable, go to <a href='https://vitty.dscvit.com'>vitty.dscvit.com</a></div>
```
Comment (iamyajat) [inline src/components/Profile.tsx:19]: href doesn't work in extensions
try using 
```js
chrome.tabs.create({ url: newURL, active: true });
```
More info: https://stackoverflow.com/questions/16503879/chrome-extension-how-to-open-a-link-in-new-tab

**Review by iamyajat — CHANGES_REQUESTED** [review]
Check the link comment and update the readme with your name as the contributor

File: src/constants/timings.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  C2: '15:00',
+  TA2: '16:00',
+  L57: '15:50',
+  TF2: '17:00',
+  // L58: '16:40',
+  TDD2: '18:00',
+  L59: '17:40',
+  // L60: '18:30',
+  V7: '19:00'
+}
```
Comment (iamyajat) [inline src/constants/timings.ts:129]: try not hardcoding the timings as it will become hard to update things if something related to it changes.

Comment (iamyajat) [reply src/constants/timings.ts:129]: use the timings that you are getting from firebase

**Review by iamyajat — CHANGES_REQUESTED** [review]

**Comment by iamyajat** [comment]
ISSUES:
- [ ] Timings do not adapt according to the timezone
- [ ] Link to the website doesn't work on click
- [ ] Use the timings fetched from firestore

OTHER CHANGES:
- [ ] Automatically open the website if no timetable is found

**Review by VisheshBansal — CHANGES_REQUESTED** [review]
Kindly fix the Linux and macOS firebase login issue. We need to fix the issue since it is bad UX.

File: src/constants/timings.ts (hunk 2 above)
Comment (KrishChatterjie) [reply src/constants/timings.ts:129]: timings are from firebase itself, these arent being used :grimacing:

File: src/components/Profile.tsx (hunk 1 above)
Comment (KrishChatterjie) [reply src/components/Profile.tsx]: sure, works now :)

**Review by VisheshBansal — APPROVED** [review]

**Comment by VisheshBansal** [comment]
LGTM @iamyajat Pls crosscheck.

**Review by iamyajat — COMMENTED** [review]
LGTM

**Review by iamyajat — APPROVED** [review]

**Pull request merged**